enumerable.rb

```diff
@@ -0,0 +1,15 @@
+require './my_enumerable'
```

Comment:
- [OPTIONAL] It will be nice to require_relative to import my_enumerable since using require_relative for files in your directory instead of require is a ruby convention. "Require should be used for external files, like gems, while require_relative should be used for referring to files within your directory."